### PR TITLE
Update README, ignore, and ci

### PR DESCRIPTION
**Pull request by reijz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +61,28 @@ cd ../bin
...
+
+Once a daemon is started, you may start the mining process
+
+```bash
+# 1. set pass phrase
+./epic-cli set-passphrase
+# 2. login
+./epic-cli login
+# 3. (optional) create transactions
+# ./epic-cli create-randomtx 50000
```
Comment (qdddddd) [inline README.md]:
```suggestion
./epic-cli create-randomtx 1
```

for creating the first registration tx. This is not optional but a must. We will soon have a command for this purpose only.

Comment (reijz) [reply README.md]: Good point

File: README.md [hunk 2]
```diff
@@ -3,80 +3,28 @@
 [![Build Status](https://travis-ci.com/EPI-ONE/epic.svg?token=xx2m4HADP8ipz4gYg3xd&branch=master)](https://travis-ci.com/EPI-ONE/epic)
 [![Coverage Status](https://coveralls.io/repos/github/EPI-ONE/epic/badge.svg?branch=master&t=OvdAhL)](https://coveralls.io/github/EPI-ONE/epic?branch=master)
 
-Epic is a pow chain that achieves an order of magnitude improvement on the throughput without sacrificing the security and decentralization.
+epic is a proof-of-work blockchain system that achieves an order of magnitude improvement on the throughput without sacrificing  security and decentralization.
```
Comment (qdddddd) [inline README.md]:
```suggestion
EPIC is a proof-of-work blockchain system that achieves an order of magnitude improvement on the throughput without sacrificing security and decentralization. 
```

Comment (reijz) [reply README.md]: I have been thinking about the same thing 
- epic
- EPIC

**Pull request merged**